### ISSUE TITLE
[REQUEST] Returning Tweets

### ISSUE THREAD

**Issue by Ryllari** [haccer/twint#86]
When running twint.Search (c), it might be possible to return a variable with the tweets found. To be able to do a search, for example:

Example:
> tw = twint.Search (c)
> tw.tweets [0] .id
> tw.tweets [0] .username

**Comment by haccer** [comment]
I've been wanting this functionality as well.

**Comment by haccer** [comment]
Yeah there's no way to cleanly do this, the `while True:` loop really makes it difficult.

**Comment by haccer** [comment]
Going to leave this open in case someone who is better than me at Python wants to bless us with it.

**Comment by pielco11** [comment]
Does this fit our needs?

```python
import twint_dev

tw = twint_dev
c = tw.Config()
c.Since = "2018-05-24"
c.Until = "2018-05-27"
c.Username = "username"
tw.run.Search(c)

print(tw.output.tweet_objects[0].id)
```
` >>> 999992115244490730`

**Comment by haccer** [comment]
@pielco11 oo. Does it keep going or only do 1 or 1 set?

**Comment by pielco11** [comment]
@haccer what do you mean? Every tweet retrieved from `Search` will be appended to the list during that phase, not matter how many tweets

**Comment by pielco11** [comment]
@Ryllari give it a try!
